acq4/devices/LightSource/LightSource.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from PyQt4 import QtCore, QtGui
+import acq4.util.Mutex as Mutex
+
+
+class LightSource(Device):
+    """Simple device which reports information of current illumination source."""
+
+    sigLightChanged = QtCore.Signal(object)
+    
+    def __init__(self, dm, config, name):
+        Device.__init__(self, dm, config, name)
+        self.lightsourceconfig = config.get('sources', config)
+
+        self.sourceState = {}
+        self.lock = Mutex.Mutex()
+        self.sigLightChanged.connect(self.lightChanged)
+
+
+    def describe(self):
+        self.description = []
+
+        for name, conf in self.lightsourceconfig.iteritems():
+            if (name=="leds"):
+                for k, v in conf.iteritems():
+                    name = k
+                    desc = {'name':name}
+
+                    for key, value in v.iteritems():
```

Comment:
It looks like this block from 30-46 basically just copies a few keys from a dict. Can this be condensed?